docs/sources/risklist/index.md

```diff
@@ -0,0 +1,42 @@
+# Risklist
+
+If you would like to generate a list of predictions on already-trained Triage model with new data, you can use the 'Risklist' module.
+
+## Examples
+Both examples assume you have already run a Triage Experiment in the past, and know these two pieces of information:
+1. A `model_id` from a Triage model that you want to use to generate predictions
+2. An `as_of_date` to generate your predictions on.
+
+### CLI
+`triage risklist <model_id> <as_of_date>`
+
+Example:
+`triage risklist 46 2019-05-06`
+
+The risklist will assume the current path to be the 'project path' to find models and write matrices, but this can be overridden by sending the `--project-path` option.
+
+### Python
+
+The `generate_risk_list` function from the `triage.risklist` module can be used similarly to the CLI, with the addition of the database engine and project storage as inputs.
+```
+from triage.risklist generate generate_risk_list
```

Comment:
Seems like this should be `from triage.risklist import generate_risk_list`?

Reply:
Yep